README.md

```diff
@@ -24,18 +24,24 @@ Support multiple data-sources.
 ![Covid-19 Recovered](https://covid19-badges.herokuapp.com/recovered/latest)
 ![Covid-19 Deaths](https://covid19-badges.herokuapp.com/deaths/latest)
 
+## New York Times is now available as a source!
+
+**Specify source parameter with ?source=nyt. NYT also provides a timeseries! To view timelines of cases by US counties use ?source=nyt&timelines=1**
```

Comment:
I would use a boolean for the example. Even though `1` will work, I think an explicit `true` is more clear.
`source=nyt&timelines=true`